example/server-hash.mjs

```diff
@@ -0,0 +1,33 @@
+import fastify from 'fastify'
+import fastifyStatic from '../index.js'
+
+const server = fastify()
+
+await server.register(fastifyStatic, {
+  root: new URL('./public', import.meta.url).pathname,
+  prefix: '/assets/',
+  hash: true,
```

Comment:
Sorry to be annoying about names, this is take-it-or-leave-it feedback, but just reading this code without reading the readme, I feel like the option `hash: true` could mean a variety of different things. I'd probably guess it means "
"add an `e-tag` header", not that it allows fancy path building for CDN busting hashes.